assets/stylesheets/common/discourse-reactions.scss

```diff
@@ -190,84 +190,77 @@ html.discourse-reactions-no-select {
   display: inline-flex;
   justify-content: flex-end;
 
-  &:not(.has-reacted) {
-    .discourse-reactions-reaction-button {
-      &:hover {
+  &.has-reacted {
+    .discourse-reactions-double-button:hover {
+      background: $primary-low;
+    }
+
+    &.has-used-main-reaction {
+      .discourse-reactions-reaction-button .reaction-button .d-icon {
+        color: $love;
+      }
+    }
+
+    &.can-toggle-reaction {
+      .discourse-reactions-reaction-button:hover .reaction-button {
         .d-icon {
-          color: $love;
+          color: $primary-low-mid;
         }
 
-        background: $love-low;
+        background: $primary-low;
       }
-    }
-  }
 
-  &.has-reacted {
-    .discourse-reactions-reaction-button {
-      .d-icon {
-        color: $love;
+      .discourse-reactions-reaction-button {
+        cursor: pointer;
       }
     }
-  }
 
-  &.has-used-main-reaction {
-    .d-icon {
-      color: $love;
-    }
+    &:not(.can-toggle-reaction) {
+      .discourse-reactions-reaction-button:hover .reaction-button {
+        background: $primary-low;
+      }
 
-    &.can-toggle-main-reaction {
+      .discourse-reactions-reaction-button button,
```

Comment:
what about this one ?

Reply:
Sorry for this, I added a new commit for this